.travis.yml

```diff
@@ -28,7 +27,7 @@ script:
     if [[ "$TRAVIS_RUST_VERSION" == nightly* ]]; then
       (cd examples && ./test_nightly)
     else
-      (cd examples && ./test_stable)
+      (cd examples && TRAVIS_RUST_VERSION=$TRAVIS_RUST_VERSION./test_stable)
```

Comment:
Somehow I'm unable to get this right... I tried `[...] && env TRAVIS_RUST_VERSION=$TRAVIS_RUST_VERSION ./test_stable` but the variable is lost in the script. (even when only adding a space `[...] && TRAVIS_RUST_VERSION=$TRAVIS_RUST_VERSION ./test_stable`)